SPEC.md

```diff
@@ -101,7 +101,9 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
                     // * r - allows container to read from the specified device.
                     // * w - allows container to write to the specified device.
                     // * m - allows container to create device files that do not yet exist.
-                    "permissions": "<permissions>" (optional)
+                    "permissions": "<permissions>" (optional),
+                    "uid": <int> (optional),
+                    "gid": <int> (optional)
```

Comment:
Do we expect this to also contain the major, minor, and type?

Reply:
+1

Reply:
done